lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -220,19 +225,19 @@ export default function SubmitGradeForm({ student }) {
       {!!submitGradeError && (
         <ErrorDialog
           title="Submit Grade Error"
-          error={{ message: submitGradeError }}
+          error={submitGradeError}
           onCancel={() => {
-            setSubmitGradeError('');
+            setSubmitGradeError(null);
           }}
           cancelLabel="Close"
         />
       )}
       {!!fetchGradeError && (
         <ErrorDialog
           title="Fetch Grade Error"
```

Comment:
Can you update this title to "Unable to fetch grade" while you're improving the error info.